packages/salesforcedx-vscode-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -258,15 +261,20 @@ export class ApexReplayDebug extends LoggingDebugSession {
     args: DebugProtocol.SetBreakpointsArguments
   ): void {
     response.body = { breakpoints: [] };
-    if (args.source.path && args.lines) {
+    if (args.source.path && args.lines && args.breakpoints) {
       this.log(
         TRACE_CATEGORY_BREAKPOINTS,
         `setBreakPointsRequest: path=${args.source
           .path} lines=${args.lines.join(',')}`
       );
       const uri = this.convertClientPathToDebugger(args.source.path);
       this.breakpoints.set(uri, []);
-      for (const lineArg of args.lines) {
+      // While processing the breakpoints, if there's a conditional breakpoint
+      // then create a checkpoint. This requires checking the breakpoint condition
+      // for the appropriate args.lines item being processed.
+      let i: number;
```

Comment:
This can be included in the for statement: `for (let i = 0; ...)`

Reply:
Done.